### ISSUE TITLE
Backup dlg immediately after upgrading wallet

### ISSUE THREAD

**Issue by concordium-cl** [Concordium/concordium-reference-wallet-android#83]
**Description**
After upgrading to a new version, the user should be prompted to backup. Most current users will be in that situation, without necessarily creating an account immediately after upgrading.

Text and buttons:

```
Welcome back!

Did you make a backup of your identities and accounts recently?

If you didn’t, now is a good time to do so! Concordium strongly recommends having a backup of all your identities and accounts. 
```
Left button: Not now
Right button: Make backup

![image](https://user-images.githubusercontent.com/76045015/149350574-51b62754-cdc9-414d-b85d-634db905ddb5.png)

 
**Steps to Reproduce**
- in old version, create id and account
- upgrade to new version

**Expected Result**
There should be an info box asking the user to make a backup, if he didn't do so in the old version, with two buttons "Dismiss" and "Okay" (leading to export). "Dismiss" should lead to the existing warning "are you sure?".

**Actual Result**
Warning dilaogs and "nagging box" first appear after new account creation.

**Versions**
- CCD testnet 1.2.1 (64)
- Android
- Huawei 6Ys

**Comment by concordium-cl** [comment]
@jens-concordium to provide some text.

**Comment by jens-concordium** [comment]
I updated the description with text, buttons and a screenshot of both.

**Comment by jensvesti** [comment]
Update from 1.1.0(62) -> 1.2.2(65) shows prompt
Update from 1.1.0(62) -> 1.2.3(66) shows prompt
Update from 1.2.2(65) -> 1.2.3(66) does not show prompt
First time install of 1.2.2(65) or 1.2.3(66) does not show prompt

**Comment by concordium-cl** [comment]
TestnetUpdate
1.2.1(64) --> 1.2.3(66) prompt shown
1.2.2(65) -->

**Comment by concordium-cl** [comment]
To be retested, depending on blocking issue.

**Comment by concordium-cl** [comment]
** 1.0.8 version after released version --> 1.2.3(66) newest version: prompt shown ✅
